src/components/shared/refreshBar/refreshBar.js

```diff
@@ -3,14 +3,18 @@
 import React, { Component } from 'react';
 import moment from 'moment';
 import { Btn } from 'components/shared';
+import { toDiagnosticsModel } from 'services/models';
 import { svgs } from 'utilities';
 import { DEFAULT_TIME_FORMAT } from 'components/shared/pcsGrid/pcsGridConfig';
 
 import './refreshBar.scss';
 
 export class RefreshBar extends Component {
 
-  refresh = () => !this.props.isPending && this.props.refresh();
+  refresh = () => {
+    this.props.logEvent(toDiagnosticsModel('Refresh_Clicked', {}));
```

Comment:
nit: most event names end "Click" instead of "Clicked".  rename "Click" for consistency.